bazci: fix null pointer exception

rickystewart opened the pull request:
Under some circumstances (failed builds) this test summary will be
empty. I'm not sure exactly why, but this fixes a null pointer
exception that otherwise pops up in this scenario.

Closes #93522.

Epic: None
Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/93351)
<!-- Reviewable:end -->

rickystewart committed:
bazci: fix null pointer exception

Under some circumstances (failed builds) this test summary will be
empty. I'm not sure exactly why, but this fixes a null pointer
exception that otherwise pops up in this scenario.

Epic: None
Release note: None

rickystewart commented:
Forgot to merge this. :(

rickystewart commented:
bors r=healthy-pod
